app/assets/javascripts/navbar.js

```diff
@@ -28,15 +28,19 @@ setupPage = function() {
         })
     }
 
+    $('.navbar').removeClass("navbar-fixed-top");
```

Comment:
Could you make sure the class is not even assigned to the element in the first place by removing the attributes in the first lines of `_navbar.html.erb` (`<%= nav_bar brand: ...`) instead of retroactively removing the class?